prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchemaReader.kt

```diff
@@ -9,19 +9,49 @@ import org.apache.logging.log4j.kotlin.Logging
 import java.io.File
 import java.io.InputStream
 
+/**
+ * Read a converter schema [schemaName] from a file given the root [folder].
+ * @return the validated schema
+ * @throws Exception if the schema is invalid or is of the wrong type
+ */
+fun converterSchemaFromFile(schemaName: String, folder: String? = null): ConverterSchema {
```

Comment:
Separate these into their own files. We want to keep all that logic very separate. They will be in separate folders with any and all code specific to each type of converter.  Also means the common code needs to be in some common folder.

Reply:
Separated out in latest commit